Api/Services/EmploymentService.cs

```diff
@@ -136,4 +136,87 @@ public async Task<Result> UpdateBulkEmploymentAsync(List<UpdateEmploymentRequest
         await context.SaveChangesAsync();
         return Result.Success;
     }
+
+    /// <summary>
+    /// Gets current or completed employments for the user that makes this request
+    /// </summary>
+    /// <param name="userId">ID of the user that makes the request</param>
+    /// <param name="isTerminated">parameter that tells if the search should include terminated xor ongoing employments</param>
+    /// <returns></returns>
+    [ErrorCode(null, ErrorCodes.NotFound)]
+    public async Task<Result<List<EmploymentSummaryVM>>> GetCurrentUsersEmploymentsAsync(string userId, bool isTerminated)
+    {
+        var user = await context.Users.SingleOrDefaultAsync(u => u.Id == userId);
+        if (user is null)
+        {
+            return new ValidationFailure
+            {
+                PropertyName = null,
+                ErrorCode = ErrorCodes.NotFound,
+                ErrorMessage = ErrorCodes.NotFound
+            };
+        }
+        var employments = await context.Employments
+            .Include(e => e.Restaurant)
+                .ThenInclude(r => r.Tags)
+            .Include(e => e.Restaurant)
+                .ThenInclude(r => r.Reviews)
+            .Include(e => e.Restaurant)
+                .ThenInclude(r => r.Logo)
+            .Where(e => e.EmployeeId == userId)
+            .ToListAsync();
+
+        if (isTerminated == true)
+        {
+            employments = employments.Where(e => e.DateUntil != null).ToList();
```

Comment:
Masz warunek na odwrót

Reply:
Warunek jest dobrze, ponieważ jak employment jest zakończony to ma jakąś datę zakończenia umowy, natomiast jak nie jest zakończony to data zakończenia nie była nigdy ustawiania, więc jest null

Reply:
A rzeczywiście, masz rację